.gitignore

```diff
@@ -1,5 +1,4 @@
 # Build Folders (you can keep bin if you'd like, to store dlls and pdbs)
-[Bb]in/
```

Comment:
These were ignoring the `bin` directory in the new local npm module. I thought it was better to remove them rather than override include the `bin` directory. No other new files seem to have been added from this